tests/testthat/test-epix_slide.R

```diff
@@ -0,0 +1,47 @@
+library(dplyr)
+library(rlang)
+
+ea <- archive_cases_dv_subset$clone()
+
+test_that("epix_slide only works on an epi_archive",{
+  expect_error(epix_slide(data.frame(x=1)))
+})
+
+test_that("epix_slide works as intended",{
+  x2 <- ea$clone()$DT %>%
```

Comment:
Is it possible to construct x2 from scratch in case we change `archive_cases_dv_subset`, and to make this more readable?  (`tribble` sometimes helps make this easier.  Or `tribble` + `unnest`, something like `tibble::tribble(~v, ~t, 5, 1:4, 6, 1:5) %>% tidyr::unnest(t)`)